### PR TITLE
Update MinVar to 2.1

### PR DESCRIPTION
**Pull request by ozagordi**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.

### CODE REVIEW

**Comment by ozagordi** [comment]
Hi.
This is strange. Only the Macos build reports a [MD5 mismatch](https://travis-ci.org/bioconda/bioconda-recipes/jobs/318152605#L361). But the Linux build doesn't, and my fork [Travis build](https://travis-ci.org/ozagordi/bioconda-recipes/builds/318142941) passes on both.

Any idea?

File: recipes/minvar/meta.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
   name: minvar
-  version: "2.0"
+  version: "2.1"
 
 build:
-  number: 1
+  number: 0
   # minvar written for python 3.x only, so skip py27
   # setuptool_scm_git_archive not available in py36
   skip: True # [py27 or py36]
```
Comment (bgruening) [inline recipes/minvar/meta.yaml]: can you remove the py36 as well, I think this should work now

**Comment by bgruening** [comment]
I have restarted the builds, let's see.

**Comment by druvus** [comment]
I think the mismatch might be caused by missing filename.

**Comment by druvus** [comment]
Thanks @ozagordi

**Comment by ozagordi** [comment]
Thank you all!

**Pull request merged**